### ISSUE TITLE
Create file store

### ISSUE THREAD

**Issue by thewilkybarkid** [libero/libero#133]
Create an implementation of the Libero file API.

[Awaiting specification]

## Notes

- Requests can be handled synchronously for now.

## Related 

- Blocked by https://github.com/libero/libero/issues/127 (API spec).

**Comment by giorgiosironi** [comment]
The object store persists the images in a publicly available location like an S3 bucket (not a responsibility of this ticket, but for clarity)

Assumption: there is no preview of articles yet
Assumption: there is no versioning (version `1` is overwritten again and again)

**Comment by thewilkybarkid** [comment]
No longer needed.
